### DISCUSSION TITLE
No project available in Octernship.

### DISCUSSION THREAD

**Discussion by sagar-io** [community/community#53474] [GitHub Education]
Why there's no project available in Octernship?

![image](https://user-images.githubusercontent.com/86215447/233769047-97894960-ff2a-43c4-a0b2-e3df074dc826.png)

**Comment by saaxena** [comment]
Because all the projects had the submission deadline between 25th march and 15 th april . Now the results will be announced. I think you are late 😔
If you find this comment valuable mark it as answered .

**Reply by sagar-io** [reply]
@saaxena  I thought it was like other internship platforms where projects get listed continuously that's why I asked. 
I was not eligible before (lack of contributions) but I think now I am.

**Reply by firoz3130** [reply]
I also want to know when new projects will be coming. Because in one of the discussions they have mentioned that new projects will be coming soon.
